jina/drivers/__init__.py

```diff
@@ -243,16 +243,10 @@ def __getstate__(self) -> Dict[str, Any]:
         return d
 
 
-class BaseRecursiveDriver(BaseDriver):
-    """A Driver to traverse a set of Documents with a specific path.
-
-    :param traversal_paths: The describes the leaves of the document tree on which _apply_all are called
+class RecursiveMixin(BaseDriver):
```

Comment:
I would rename it later, because for some Drivers we may still need a different mixin (kind of a ContextAwareRecursiveMixin).

but anyhow the rename can be done later